Package.resolved

```diff
@@ -1,5 +1,14 @@
 {
   "pins" : [
+    {
```

Comment:
I'm going to have vendorize something else soon though because the whole is pretty massive and I'm not sure they support cocoapods anymore. Fortunatley this one is tiny.